### PR TITLE
Added hearthstone card images

### PR DESCRIPTION
**Pull request by jonathanhawleypeters**
@jesterswilde have a look-see

Edit: forgot to update cardViewQuery, fixed

### CODE REVIEW

File: front-end/src/components/app.js [hunk 1]
```diff
@@ -94,6 +94,9 @@ class App extends Component {
 				<main className='meta-container'>
 					<div className='tile is-ancestor is-vertical'>
 						<PropsRoute path='/' component={CardSearch} appData={appData} />
+						<div>
+							©2014 Blizzard Entertainment, Inc. All rights reserved. Heroes of Warcraft is a trademark, and Hearthstone is a registered trademark of Blizzard Entertainment, Inc. in the U.S. and/or other countries. Meta -ls is not affiliated or associated with or endorsed by Hearthstone® or Blizzard Entertainment, Inc.
+						</div>
```
Comment (jesterswilde) [inline front-end/src/components/app.js]: I thought this was going to be a footer.

File: front-end/src/components/card.js [hunk 2]
```diff
@@ -8,16 +8,16 @@ const typesComparator = (a, b) => {
...
   const isClickable = (transitioner)? ' clickable': ''; 
   return (
     <section className='tile no-padding'>
       <div className='tile is-child is-4'>
         <figure className='resultsCardImage'>
           <img 
-            src={`https://cards.nyc3.digitaloceanspaces.com/${name}.jpg`}
-            alt='card back filler'
+            src={`https://cards.nyc3.digitaloceanspaces.com/hearthstone/${dbfId}.png`}
+            alt='Image missing'
```
Comment (jesterswilde) [inline front-end/src/components/card.js]: This should say the name of the card or something similar. This is what happens when you hover the card. It also is what a screen-reader reads.

File: front-end/src/components/cardList.js [hunk 3]
```diff
@@ -94,8 +94,8 @@ class CardList extends Component {
                   <Card key={card.name} card={card} transitioner={()=> toCard(this.props, card.name)} />
                 </div>
                 <div className='tile   is-parent'>
-                  <div className = 'tile is-child is-size-3'>
-                    Card Rating: {card.totalRating || 'Fucking nothing'}
+                  <div className = 'tile is-child is-size-4'>
+                    {card.totalRating ? "Card Rating:" + card.totalRating : 'Be the first to rate this card'}
```
Comment (jesterswilde) [inline front-end/src/components/cardList.js]: Better, but this whole section should probably get a re-work. We can stick more info into this third. 

Also, how do we want to display the rating (ideally) ? Stars, numbers, mana crystals?

File: front-end/src/components/app.js (hunk 1 above)
Comment (jonathanhawleypeters) [reply front-end/src/components/app.js]: When would a used ever see a footer? Right now, the final ground of cards repeats infinitely.

This message needs to live somewhere else, I really just put it there as a reminder/cya measure if we deploy to heroku

File: front-end/src/components/card.js (hunk 2 above)
Comment (jonathanhawleypeters) [reply front-end/src/components/card.js]: Ah, thanks. I will change it.

File: front-end/src/components/cardList.js (hunk 3 above)
Comment (jonathanhawleypeters) [reply front-end/src/components/cardList.js]: I really put to many different changes in this pr :(

Part of me wants to display a number, and I feel it's most justifiable in this view. Another part of me wants consistency, which I think would imply a single language for representing ratings.

Let's open an issue to figure out ratings ui, and another issue to determine what info displays in the rightmost third of a card in this list.

Comment (jonathanhawleypeters) [reply front-end/src/components/cardList.js]: I've entered the issues that deal with reworking ratings and displaying more info in the results list

**Pull request merged**